Gemfile

```
@@ -119,5 +115,10 @@ group :test do
   gem "webmock"
 end
 
+group :development, :test, :review do
```

> The task in this PR will be run on QA and Staging to set up their DBs with example data.

Does that mean we need something like?

```suggestion
group :development, :test, :review, :qa, :staging do
```

---

Yeah. I'm not sure about that. I don't know if it's just a heroku think that it tries to interpret all of the rake tasks. When I set the env to <insert-env-here> on my machine rake -P runs fine as is. I think it might be just on heroku. I'll deploy it to qa this morning and see. The other option is to just move those gems out of the groups altogether :shrug: